src/components/Header/headerStyles.ts

```diff
@@ -1,20 +1,60 @@
 import styled from "styled-components";
 import { NavLink } from "react-router-dom";
+import burger from "../../img/burger.png";
 
 interface FlagsWrapperProps {
   "data-hovered": boolean;
 }
 
-export const Div = styled.div`
-  position: relative;
+export const MobileNav = styled.div`
+  display: none;
+  @media (max-width: 768px) {
+    display: block;
+  }
+`;
+
+export const MobileNavList = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  margin-top: 10px;
+
+  background: #2c3e50;
+  position: absolute;
+  top: 70px;
+  left: 0;
+  width: 100%;
+  z-index: 1000;
```

Comment:
do we need this?

Reply:
yes, these are styles that control the position of the mobile menu